inst/tests/tests.Rraw

```diff
@@ -2912,7 +2912,7 @@ test(1034, as.data.table(x<-as.character(sample(letters, 5))), data.table(V1=x))
 
   # segfault of unprotected var caught with the help of address sanitizer
   set.seed(1)
-  val = sample(c(1:5, NA), 1e5L, TRUE)
+  val = sample(c(1:5, NA), 1e4L, TRUE)
```

Comment:
confirmed this test is valid also using 1e4L size

Reply:
Great. Knowing that makes it easy to approve -- thanks.